warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/CompositeIngest.java

```diff
@@ -69,8 +69,11 @@ static boolean isOverloadedCompositeField(Multimap<String,String> compositeField
     }
 
     static boolean isOverloadedCompositeField(Collection<String> compFields, String compositeFieldName) {
-        if (compFields != null && !compFields.isEmpty())
-            return compFields.stream().findFirst().get().equals(compositeFieldName);
+        if (compFields != null && !compFields.isEmpty()) {
+            if (compFields.stream().findFirst().isPresent()) {
```

Comment:
Intellij is pointing out that this if statement is always true due to the check in the previous if statement.  let's remove the redundant if statement.

Reply:
fixed.